sigle/src/modules/editor/HintsCarousel.tsx

```diff
@@ -0,0 +1,151 @@
+import {
+  ArrowLeftIcon,
+  ArrowRightIcon,
+  MixerHorizontalIcon,
+  TrashIcon,
+} from '@radix-ui/react-icons';
+import Image from 'next/image';
+import { useEffect, useState } from 'react';
+import { RoundPlus } from '../../icons';
+import { styled } from '../../stitches.config';
+import { Box, Flex, IconButton, Text } from '../../ui';
+
+const ImgWrapper = styled('div', {
+  width: 600,
+  height: 350,
+  position: 'relative',
+  mx: 'auto',
+  boxShadow: '0 0 0 1px $colors$gray7',
+  borderRadius: '$1',
+  overflow: 'hidden',
+});
+
+export const HintsCarousel = () => {
```

Comment:
I was wondering, can we display the content only of the current selected tab? That way the GIF will be loaded only when you display the hint, also it means it starts only when you navigate to it and does not continue to play in the background. Wdyt?

Reply:
I just checked in dev tools when the images are loading, and wondering if this is what we want to happen? (GIF loads only when we click on Tab) or does it continue playing after we change the Tab and we need to prevent that?

https://user-images.githubusercontent.com/65421744/157079873-22fcdf5b-fc09-4a72-969b-81c8ddd5e9ba.mov

Reply:
Okay indeed I was confused as it continued playing in the background, if it starts when you load the tab then all good 👍